### PR TITLE
Decompile a bunch of small files

### PR DESCRIPTION
**Pull request by Random06457**
mtxuty-cvt.c
padutils.c
absf.c
guS2DInitBg.c
osAiSetFrequency.c
osSetTimer.c
osSpTaskYield.c
osSpTaskYielded.c
osStopTimer.c
rotate.c (ported from mm)
sqrt.c

### CODE REVIEW

File: src/libultra_code/osAiSetFrequency.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include <global.h>
+#include <ultra64/hardware.h>
+
+s32 osAiSetFrequency(u32 frequency) {
+    u32 dacrate;
+    u8 bitrate;
```
Comment (fig02) [inline src/libultra_code/osAiSetFrequency.c:6]: this is supposed to read as DAC Rate right? i would use camel case for both of these to make them easier to read. idk if bitrate is usually one word, but if its not maybe it should be changed as well

Comment (Random06457) [reply src/libultra_code/osAiSetFrequency.c]: I think bitrate is usually one word.

File: src/libultra_code/guS2DInitBg.c [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    shift = (6 - bg->b.imageSiz);
+    if (bg->b.imageLoad == G_BGLT_LOADBLOCK) {
+        bg->b.tmemW = bg->b.imageW >> shift;
+        bg->b.tmemH = (tmem / bg->b.tmemW) * 4;
+        bg->b.tmemSizeW = bg->b.tmemW * 2;
+        bg->b.tmemSize = bg->b.tmemH * bg->b.tmemSizeW;
+        bg->b.tmemLoadSH = (bg->b.tmemSize >> 1) - 1;
+        bg->b.tmemLoadTH = (0x7FF / bg->b.tmemW) + 1;
+    } else // G_BGLT_LOADTILE
+    {
```
Comment (Roman971) [inline src/libultra_code/guS2DInitBg.c:19]:
```suggestion
    } else { // G_BGLT_LOADTILE
```

File: src/libultra_code/sqrt.c [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#include <math.h>
+#include <global.h>
```
Comment (Roman971) [inline src/libultra_code/sqrt.c:2]:
```suggestion
#include <global.h>
```

File: Makefile [hunk 4]
```diff
@@ -109,6 +109,12 @@ build/src/libultra_boot_O2/%.o: CC := $(CC_OLD)
 build/src/libultra_code/%.o: CC := python3 tools/asm_processor/build.py $(CC_OLD) -- $(AS) $(ASFLAGS) --
 build/src/code/jpegutils.o: CC := python3 tools/asm_processor/build.py $(CC_OLD) -- $(AS) $(ASFLAGS) --
 
+build/src/libultra_code/sqrt.o: OPTFLAGS := -O2 -g3
+build/src/libultra_code/absf.o: OPTFLAGS := -O2 -g3
+# Todo: split libultra_code into libultra_code_O1 / libultra_code_O2
+build/src/libultra_code/osSetTimer.o: OPTFLAGS := -O1
+build/src/libultra_code/osStopTimer.o: OPTFLAGS := -O1
```
Comment (Roman971) [inline Makefile:116]: These should be grouped to the other OPTFLAGS definitions above.

**Pull request merged**